ajax/libs/xterm/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name":"xterm",
+  "filename":"xterm.min.js",
+  "version":"2.3.2",
+  "description":"Full xterm terminal, in your browser",
+  "homepage":"https://xtermjs.org/",
+  "repository":{
+    "type":"git",
+    "url":"git+https://github.com/sourcelair/xterm.js"
+  },
+  "keywords":[
+    "xterm",
+    "terminal",
+    "browser-tty"
+  ],
+  "author":"SourceLair",
+  "license":"MIT",
+  "autoupdate":{
+    "source":"git",
+    "target":"https://github.com/sourcelair/xterm.js.git",
```

Comment:
@jagadeeshv16 Please follow the target format in [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md). Let's use `git://github.com/sourcelair/xterm.js.git`.